test/testdata/app-scenarios/petset.yaml

```diff
@@ -492,5 +402,95 @@ items:
     phase: Running
     podIP: 172.17.0.4
     startTime: 2016-07-27T03:00:58Z
+- apiVersion: apps/v1alpha1
+  kind: PetSet
+  metadata:
+    creationTimestamp: 2016-07-21T15:53:09Z
+    generation: 3
+    labels:
+      app: mysql
+    name: mysql
+    namespace: default
+    resourceVersion: "6790"
+    selfLink: /apis/apps/v1alpha1/namespaces/default/petsets/mysql
+    uid: 3900c985-4f5b-11e6-b8a1-080027242396
+  spec:
+    replicas: 3
+    selector:
+      matchLabels:
+        app: mysql
+    serviceName: galera
+    template:
+      metadata:
+        annotations:
+          pod.alpha.kubernetes.io/init-containers: '[{"name":"install","image":"gcr.io/google_containers/galera-install:0.1","args":["--work-dir=/work-dir"],"resources":{},"volumeMounts":[{"name":"workdir","mountPath":"/work-dir"},{"name":"config","mountPath":"/etc/mysql"}],"terminationMessagePath":"/dev/termination-log","imagePullPolicy":"Always"},{"name":"bootstrap","image":"debian:jessie","command":["/work-dir/peer-finder"],"args":["-on-start=\"/work-dir/on-start.sh\"","-service=galera"],"env":[{"name":"POD_NAMESPACE","valueFrom":{"fieldRef":{"apiVersion":"v1","fieldPath":"metadata.namespace"}}}],"resources":{},"volumeMounts":[{"name":"workdir","mountPath":"/work-dir"},{"name":"config","mountPath":"/etc/mysql"}],"terminationMessagePath":"/dev/termination-log","imagePullPolicy":"IfNotPresent"}]'
```

Comment:
Break the `[{...}]` string into lines. E.g. https://github.com/kubernetes/contrib/blob/master/pets/zookeeper/zookeeper.yaml#L35